src/Components/DnDLayout/GridTile.tsx

```diff
@@ -189,13 +200,13 @@ const GridTile = ({ widgetType, isDragging, setIsDragging, setWidgetAttribute, w
               )}
               {hasHeader && isLoaded && (
                 <FlexItem>
-                  <Button
-                    className="pf-v5-u-font-weight-bold pf-v5-u-font-size-xs pf-v5-u-p-0"
-                    variant="link"
-                    onClick={() => window.open(headerLink.href, '_blank')}
-                  >
-                    {headerLink.title}
-                  </Button>
+                  {headerLink.href && (
+                    <Link to={widgetLink(headerLink.href)}>
```

Comment:
You can't mix client-side routing with classic HTML routing. You have to pick a link component based on what the target is.

```jsx
// for origin console.redhat.com

const href = https://www.google.com/

<Link to={href}>...</Link>
// actual link: console.redhat.com/https://www.google.com/
```

You have to decide which type of a link you are supposed to use.

Reply:
I'm not sure I understand. Do I pick a different routing component based on whether its external or not?

Reply:
Exactly. For external links, you use the plain HTML tags.